dependencies.gradle

```diff
@@ -31,10 +31,6 @@ ext {
   fullTestRun = (project.hasProperty('square.fullTestRun') ?
       project.getProperty('square.fullTestRun') : 'true').toBoolean()
 
-  if (ci && !fullTestRun) {
```

Comment:
Is it possible to keep this check but add an exclusion for when we're running on windows? 🤔 

The check seems like a useful safe-guard to make sure any future changes to not run all tests are intentionally committed